app/src/main/java/fr/free/nrw/commons/contributions/ContributionsListFragment.java

```diff
@@ -236,6 +244,12 @@ private void setListeners() {
       controller.initiateGalleryPick(getActivity(), true);
       animateFAB(isFabOpen);
     });
+
+    fabCustomGallery.setOnClickListener(v -> {
```

Comment:
This can be annotated with Butterknife's OnClick

Reply:
Done.